app/views/govuk_publishing_components/components/docs/input.yml

```diff
@@ -30,6 +30,13 @@ examples:
         text: "What is your email address?"
       name: "address"
       type: "email"
+  numeric_input:
+    description: If input is set to `type="number"` we setup the component up as described in the [Design System guidance](https://design-system.service.gov.uk/components/text-input/#numbers) adding `input-mode` and `pattern`. These values can be overriden if necessary.
```

Comment:
I think this should be `inputmode` not `input-mode`? Also typo: `overridden`.

Reply:
Also `setup the component up`.

Reply:
Sorry, evening rush. Fixed now.